modules/ossm-install-elasticsearch.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+// * serverless/installing-openshift-serverless.adoc
+
+[id="ossm-operator-install-elasticsearch_{context}"]
+= Installing the Elasticsearch Operator
```

Comment:
As of OSSM 1.0.2, the Elasticsearch Operator will automatically be installed (if not already present), via the Jaeger operator, when the Service Mesh operator is installed. So not sure we need this as a separate doc now?

Reply:
I was unaware of that change.